### PR TITLE
Add 0.16 release date

### PR DESCRIPTION
**Pull request by miguelusque**
cuDF 0.16 release date updated.

### CODE REVIEW

**Review by galipremsagar — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,7 +36,7 @@
 - PR #6575 Fix JNI RMM initialize with no pool allocator limit
 
 
-# cuDF 0.16.0 (Date TBD)
+# cuDF 0.16.0 (23 Oct 2020)
```
Comment (galipremsagar) [inline CHANGELOG.md:39]: Shouldn't it be the 21st?
https://github.com/rapidsai/cudf/blob/main/CHANGELOG.md#cudf-0160-21-oct-2020

**Comment by miguelusque** [comment]
Hi!

I am not sure if it is the 21st or 23rd.

I used 23rd which was the day when Josh publicly announced it in Twitter. :-)

**Comment by galipremsagar** [comment]
I think based on [this](https://docs.rapids.ai/releases/schedule) it is 21st, @dillon-cullinan @raydouglass can correct me here.

**Comment by galipremsagar** [comment]
There appears to be an auto-merge to fix this: https://github.com/rapidsai/cudf/pull/6576

**Comment by mike-wendt** [comment]
> There appears to be an auto-merge to fix this: #6576

Yes please correct the auto-merge PR using the docs in the link

**Comment by mike-wendt** [comment]
Duplicate of #6576

**Pull request closed without merging**